.travis.yml

```diff
@@ -15,6 +15,8 @@ jobs:
       env: TOXENV=flake8
     - name: "Check documentation with darglint"
       env: TOXENV=darglint
+    # - name: "Check typing with MyPy"
+    #   env: TOXENV=mypy
```

Comment:
is this commented out on purpose?

Reply:
Yes. we're not ready for this kind of power. it's hundreds of lines of output right now...